### PR TITLE
refactor: use frontend-component-header, move away from old studio styles

### PR DESCRIPTION
**Pull request by brian-smith-tcril**
this PR has all the changes from https://github.com/openedx/frontend-app-library-authoring/pull/69

the differences are:
* the commits are squashed (the WIP pr has a lot of unhelpful commit messages, as well as commits that aren't at working stages)
* the `package.json` points to the `frontend-component-header` from npm instead of from my fork

### CODE REVIEW

File: src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx [hunk 1]
```diff
@@ -136,51 +136,52 @@ testSuite('<LibraryAuthoringPageContainer />', () => {
     expect(fetchLibraryBlockView.fn).toHaveBeenCalledTimes(1);
   });
 
-  it('Fetches block LTI URL to clipboard', async () => {
-    const library = libraryFactory({ allow_lti: true });
-    const blocks = makeN(blockFactoryLine([], { library }), 2);
+  // todo: figure out how LTI stuff works
```
Comment (brian-smith-tcril) [inline src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx:139]: @arbrandes i'm not sure how to test this. i'll gladly bring back the buttons and uncomment these tests once i know how to set up a library to have LTI stuff

Comment (arbrandes) [reply src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx:139]: Let's push the LTI stuff back until the rest is done - maybe even further.  It requires backend fixes before we can revisit the frontend properly.

**Comment by arbrandes** [comment]
@brian-smith-tcril, tested it, and it (still) works great!  Can you please rebase on master?

Also, it would be great if we didn't have to merge with a failing `codecov/project`.  I know, it's just a -0.46% diff at time of writing, but it wouldn't be a great example to set.  (There's a separate issue in the [spring cleaning checklist](https://github.com/openedx/frontend-app-library-authoring/issues/71) for tests, so we could think of this as a start-off point.)

**Comment by brian-smith-tcril** [comment]
@arbrandes this is ready for eyes again

i updated the [header PR](https://github.com/openedx/frontend-component-header/pull/271) and created a [testing branch](https://github.com/brian-smith-tcril/frontend-app-library-authoring/tree/testable-header-component-usage) that updates the `package.json` to point to a [branch of the forked header with dist checked in](https://github.com/brian-smith-tcril/frontend-component-header/tree/studio-header-component-dist)

**Review by arbrandes — APPROVED** [review]
@brian-smith-tcril, merged the frontend-component-header PR.  Tested it in the process of testing this (via your test branch).  Looks great!  Let me know if we can merge it as is, or if we need a version bump for `frontend-component-header`.

**Comment by brian-smith-tcril** [comment]
@arbrandes a version bump on the header repo would be ideal. just for peace of mind i'd like to verify everything is working in the code we're about to merge before hitting the button

**Pull request merged**